heliopy/data/util.py

```diff
@@ -176,6 +181,40 @@ def units_attach(data, units):
     return timeseries_data
 
 
+def cdf_units(cdf_, keys=None):
+    """
+    Takes the CDF File and the required keys, and finds the units of the
+    selected keys.
+
+    Parameters
+    ----------
+    cdf_ : cdf
+        Opened cdf file
+    keys : :class 'dict'
```

Comment:
I think you can just do `keys : dict` and sphinx will be clever enough to pick up the link to dict.